untrusted/test_enclave/Cargo.toml

```
@@ -17,3 +17,4 @@ cc = "1.0.73"
 bindgen = "0.60.1"
 rsa = "0.6.1"
 rand = "0.8.5"
+mc-sgx-core-build = { path = "../../core/build" }
```

Nit: Add a newline to maintain separate sorted groups
```suggestion

mc-sgx-core-build = { path = "../../core/build" }
```